lexicons/app/bsky/notification/putPreferencesV2.json

```diff
@@ -0,0 +1,83 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.notification.putPreferencesV2",
```

Comment:
I assume this replaces the functionality of `app.bsky.notification.putPreferences`.  Is the `priority` setting from that endpoint modeled here somewhere?